packages/next/server/api-utils.ts

```diff
@@ -63,6 +63,26 @@ export async function apiResolver(
       )
     }
 
+    let contentLength = 0
+    const writeData = apiRes.write
+    const endResponse = apiRes.end
+    apiRes.write = (...args) => {
+      contentLength += Buffer.byteLength(args[0])
+      writeData.apply(apiRes, args)
```

Comment:
```suggestion
      return writeData.apply(apiRes, args)
```
Looks like the types want us to return here and the same on `endResponse.apply`